src/Apis/EndPoints.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+const baseURL = "https://upskilling-egypt.com:3005/api";
+export const apiClient = axios.create({
+  baseURL,
+});
+
+apiClient.interceptors.request.use((config) => {
+  const token = localStorage.getItem("userToken");
+  if (token) {
+    config.headers.Authorization = `Bearer ${token}`; // Make sure to include 'Bearer ' if your API requires it
+    console.log(token);
+  }
+  return config;
+});
+
+export const AUTHENTICATION_URLS = {
+  regitser: `${baseURL}/auth/register`,
```

Comment:
since we have the base url in the apiClient, we don't need to include it here again.

Reply:
Only the change password in the registeration requires the token, for that reason we can make two axios  instances: public one and private one. The public one will be initialized with base url and the private one will be initialized with both the base url and the token in the interceptors.